app/javascript/Components/__tests__/dropdown_menu.test.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import {render, screen, fireEvent} from "@testing-library/react";
+import {DropDownMenu} from "../Result/dropdown_menu";
+
+describe("DropDownMenu", () => {
+  const mockItems = [
```

Comment:
These tests are good, but please make the following additions:

1. Add an item without a deduction to `mockItems` (to make sure that's rendered correctly)
2. Add a test to make sure that when an item is clicked, the passed in mock function is triggered.